### ISSUE TITLE
Icon for lonestar in atek does nothing

### ISSUE THREAD

**Issue by knownasilya** [atek-cloud/lonestar#1]
Not sure if this is by design, but it seems weird to show the icon if it is basically a home button.

It takes the user to lonestar.localhost, which from a users perspective does nothing since it's the default ui.

**Comment by pfrazee** [comment]
Yeah known issue. I had to recently make lonestar a user-installed app, and consequently it's showing up in the app icons.
